Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.yml

```diff
@@ -9,30 +9,32 @@ configuration:
   required: true
```

Comment:
polling : true 
should be added to these commands, as well as the first_run_flag argument